src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -145,14 +129,8 @@ const FooterExtraActions = styled.div`
   flex-direction: row;
   justify-content: center;
   align-items: center;
-  margin-top: 1.5rem;
-
-  > * {
-    margin-right: 1.375rem;
-    :last-child {
-      margin-right: 0;
-    }
-  }
+  margin-top: 0.75rem;
+  margin-bottom: 0.25rem;
 `
 
 export const Sidebar = React.forwardRef<HTMLDivElement, Props>(
```

Comment:
Forgot what we said if we should fix this things in components but, getting linting warnings on UseEffect,
That we should add to the dependency array.
It's a pretty easy fix.
two of them are functions and will probably not change, but
redeemedCampaigns, this one is data from a query so maybe this one should be in it?

Reply:
Added a new PR for that to isolate potential issues :) Still have to add all of them as dependencies in order to get rid of the lint warning https://github.com/HedvigInsurance/web-onboarding/pull/454